scripts/convert.R

```diff
@@ -0,0 +1,45 @@
+## script to read in mtx files from ANND experiments
+## and create SeuratObject
+##
+suppressMessages(library(devtools))
+devtools::install_github('cellgeni/sceasy')
+suppressMessages(library(sceasy))
+suppressMessages(library(Seurat))
+suppressMessages(library(plyr))
+suppressMessages(library(stringr))
+
+args <- commandArgs(trailingOnly = TRUE)
```

Comment:
You can add a check here:
```suggestion
args <- commandArgs(trailingOnly = TRUE)

if(length(args) != 1) {
   stop("Not correct number of arguments. Please supply 1 argument")
}
```